packaging_apps_helpers.md

```diff
@@ -2,85 +2,264 @@
 
 ## Shell helpers
 
-Since YunoHost 2.4 release, **new shell helpers** are available to ease packaging, in particular for common tasks like password generation, MySQL database creation…
+Since YunoHost 2.4 release, **new shell helpers** are available to ease common packaging tasks like password generation, MySQL database creation…
 
-Examples are available in the [example application](https://github.com/YunoHost/example_ynh/blob/master/scripts/install). We advise to use them. 
+Usage examples are available in the [example application](https://github.com/YunoHost/example_ynh/blob/master/scripts/install). We advise to use them.
 
 You can find them on this [repository](https://github.com/YunoHost/yunohost/blob/unstable/data/helpers.d).
 
 To use them, you need to add following lines in the shell scripts:
 ```bash
 # Source app helpers
-. /usr/share/yunohost/helpers
+source /usr/share/yunohost/helpers
 ```
 
-<!--
-<br />
+### Available helpers list (non exhaustive)
+#### Database
+
+```bash
+ynh_mysql_execute_as_root SQL DB
+```
+
+> Runs the SQL command `SQL` as database root user on `DB` database (this last argument is optional).
+
+```bash
+ynh_mysql_execute_file_as_root FILE DB
+```
+
+> Runs the SQL commands listed inside `FILE` as root user on `DB` database (this last argument is optional).
+
+```bash
+ynh_mysql_create_db DB USER PWD
+```
+> Create the `DB` database and grants all rights to `USER` (created on the fly with `PASSWORD` password).
+
+```bash
+ynh_mysql_drop_db DB
+```
+> Delete the `DB` database.
+
+```bash
+ynh_mysql_dump_db DB > ./FILE
+```
+
+> Exports the `DB` database into the `FILE` file.
+
+#### Debian packages handling
+
+```bash
+ynh_package_is_installed PACKAGE
+```
+
+> Tests if the Debian package `PACKAGE` is installed on the system.
+    > Read command output to know the result. e.g:
+> ```bash
+> if ! ynh_package_is_installed "yunohost" ; then
+>   echo "Oops, package is not installed"
+> else
+>   echo "Package is installed"
+> fi
+> ```
+
+```bash
+ynh_package_version PACKAGE
+```
+> Returns the installed version number of `PACKAGE`.
+
+```bash
+ynh_package_update
+```
+> Updates the packages list (`apt update`) in a silent and non-interactive way.
+
+**Be careful, the following commands are to be avoided when possible. Installing (and even more removing) a package without handling conflicts and dependencies. That will be improved in future Yunohost versions.**
```

Comment:
Isn't a verb lacking at the end of the middle sentence ? Like "installing a package without handling conflicts and dependencies **is not recommended**" ?